webapp/src/main/webapp/src/app/pages/importers/importers.page.ts

```diff
@@ -102,8 +103,8 @@ export class ImportersPageComponent implements OnInit {
     this.route.queryParams.subscribe(queryParams => {
       // Look at query parameters to apply filters.
       this.filterConfig.appliedFilters = [];
-      if (queryParams['name']) {
-        this.nameFilterTerm = queryParams['name'];
+      if (queryParams.name) {
```

Comment:
Do you think this is actually safe and allow Typescript compilation? `name` is not a declared property on `queryParams` object I think...

Reply:
No difference in that case